File: PTT-GC-SPA/src/app/core/models/MainPlant.ts
export class MainPlant {
  constructor() { }
  public mainPlanId: number;
  public eMocTitle: string;
  public initiativeId: number;
  public plant: string;
  public areaUnit: string;
  public typeOfChange: string;
  public mocCategory: string;
  public expireDate: Date;
  public detailOfTheChange: string;
  public initialListOfPeople: string;
  public emocNo: string;
  public mocChampion: string;
  public isMainPlant: boolean;
  public goalAchievement: string;
  public specifiedGoalAchievement: string;
  public assumptionOfGoal: string;
  public reasonForChange: string;
}

export interface MocStatus {
  status: string;
  message: string;
  moCs: any[];
}

File: PTT-GC-API/Models/EMOCs/MainPlant.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace PTT_GC_API.Models.EMOCs
{
    public class MainPlant
    {
        [Key]
        public int MainPlanId { get; set; }
        public string EMocTitle { get; set; }
        public int? InitiativeId { get; set; }
        public string Plant { get; set; }
        public string AreaUnit { get; set; }
        public string TypeOfChange { get; set; }
        public string MocCategory { get; set; }
        public DateTime? ExpireDate { get; set; }
        public string DetailOfTheChange { get; set; }
        public string InitialListOfPeople { get; set; }
        public string EmocNo { get; set; }
        public string MocChampion { get; set; }

        public bool? IsMainPlant { get; set; }
        public string GoalAchievement { get; set; }
        public string SpecifiedGoalAchievement { get; set; }
        public string AssumptionOfGoal { get; set; }
        public string ReasonForChange { get; set; }
    }
}
